inc/saifdb.h

```diff
@@ -282,22 +285,26 @@ typedef sai_status_t (*sai_get_fdb_entry_attribute_fn)(
 /**
  * @brief Remove all FDB entries by attribute set in sai_fdb_flush_attr
  *
+ * @param[in] switch_id Switch object id
  * @param[in] attr_count Number of attributes
  * @param[in] attr_list Array of attributes
  *
  * @return #SAI_STATUS_SUCCESS on success Failure status code on error
  */
 typedef sai_status_t (*sai_flush_fdb_entries_fn)(
+        _In_ sai_object_id_t switch_id,
         _In_ uint32_t attr_count,
         _In_ const sai_attribute_t *attr_list);
 
 /**
  * @brief FDB notifications
  *
+ * @param[in] switch_id Switch object id
  * @param[in] count Number of notifications
  * @param[in] data Pointer to fdb event notification data array
  */
 typedef void (*sai_fdb_event_notification_fn)(
+        _In_ sai_object_id_t switch_id,
```

Comment:
Why do we need switch_id for sai_fdb_event_notification
The event data has event notification data items
Each data item contains fdb_entry, which already contains switch_id